app/domain/logs.rb

```diff
@@ -188,6 +188,20 @@ module AuthnAzure
       )
 
     end
+
+    module AuthnGCP
+
+      ExtractingRestrictionsFromResource = ::Util::TrackableLogMessageClass.new(
+          msg: "Extracting resource restrictions from role '{0-role-id}' with prefix '{1-prefix}'",
+          code: "CONJ00039D"
+      )
+
+      ExtractedResourceRestrictions = ::Util::TrackableLogMessageClass.new(
+          msg: "{0-amount} resource restrictions were extracted",
```

Comment:
@shulifink please review

Reply:
earlier you mentioned that you think these should be upper case. Same here, no?
But why should they be uppercase?

Reply:
I think this is fine as it is

Reply:
i thought of doing that because Resource Restrictions is a thing (it actually replaces Application Identity) but i then changed my mind and prefer it lower-cased